src/app/components/InviteNewAccount.js

```diff
@@ -277,13 +277,11 @@ const InviteNewAccount = ({ teamSlug }) => (
           dbid
           name
           email
-          login
```

Comment:
These were simple fixes because the component that uses the data is currently within the same file; so I just removed the fields as they are obviously never used.